styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java

```diff
@@ -602,6 +602,31 @@ public void shouldUpdateBackfill() throws Exception {
     assertThat(storage.backfill(BACKFILL_1.id()).get().concurrency(), equalTo(4));
   }
 
+  @Test
+  public void shouldHaltBackfillReversed() throws Exception {
```

Comment:
Is there anything special when halting a reverse backfill?

Reply:
Halting goes through `retrieveBackfillStatuses` and calculating `instantsInRange`